### PR TITLE
hardening/893_add_test_for_agent_file_name

### PR DESCRIPTION
**Pull request by pcoello25**
* Partially fix issue #893 
  * Update CHANGES_NEXT_RELEASE is not neccesary.
* 100% unit tests passed:
```
Results : Tests run: 73, Failures: 0, Errors: 0, Skipped: 0
```
* Logs:
```
[ManagementInterface.handleAgentConfFileName] ------- -  Agent configuration doesn't start with 'agent_'.
[ManagementInterface.handleAgentConfFileName] ------- -  OK  - An agent configuration file not starting with 'agent_' has been detected
```
* Assignee: @frbattid

### CODE REVIEW

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java [hunk 1]
```diff
@@ -332,7 +345,7 @@ public void testPostMethodPostAValidSubscriptionV1() throws Exception {
         
         try {
             assertEquals(HttpServletResponse.SC_OK, responseWrapper.getStatus());
-            System.out.println(getTestTraceHead("[ManagementInterface.handlePostSubscription]") + " -  OK  - Valid "
+            System.out.println(getTestTraceHead("[ManagementInterface.handlePostSubscription]") + " - OK  - Valid "
```
Comment (frbattid) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: This "OK" is not aligned.

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java [hunk 2]
```diff
@@ -844,4 +858,36 @@ public void testDeleteMethodDeleteOneAgentConfigurationParameter() throws Except
...
+             throw x;		
+         } // try catch
+         
+         try {		
+             assertEquals(HttpServletResponse.SC_BAD_REQUEST, responseWrapper.getStatus());		
+             System.out.println(getTestTraceHead("[ManagementInterface.handleAgentConfFileName]") + " - "
+                     + "OK  - An agent configuration file not starting with 'agent_' has been detected");		
+         } catch (AssertionError e) {		
+             System.out.println(getTestTraceHead("[ManagementInterface.handleAgentConfFileName]") + " - "
+                     + "FAIL - There are some problems with your request");
```
Comment (frbattid) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: This message must be something like "An agent configuration file not starting with 'agent_' has not been detected".

Comment (pcoello25) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Fixed in 62872d8

File: cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java (hunk 1 above)
Comment (pcoello25) [inline cygnus-common/src/test/java/com/telefonica/iot/cygnus/management/ManagementInterfaceTest.java]: Fixed in 438ffe6

**Comment by frbattid** [comment]
LGTM

**Pull request merged**